ui/yarn.lock

```diff
@@ -3947,7 +3947,8 @@ anymatch@~3.1.1:
     picomatch "^2.0.4"
 
 "api-common-protos@link:./lib/api-common-protos":
-  version "1.0.0"
+  version "0.0.0"
+  uid ""
```

Comment:
I’m still ensure why Yarn does this. It seems to me that `version "1.0.0"` is more correct. 🤷‍♀️